test/integration/version-18.03.1/testinfra/test_docker.py

```diff
@@ -2,9 +2,9 @@
 
 
 def test_package_is_installed(Package):
-    docker = Package('docker-engine')
+    docker = Package('docker-ce')
```

Comment:
Could the package name be configurable?

Reply:
See #153 could you use `docker:pkg:name` here.

Reply:
Can't use a saltstack pillar var in test python code without non-trivial work